src/routes/amt/powerAction.ts

```diff
@@ -171,6 +174,7 @@ function powerStateChange (uuid, action, amtstack, req, res): void {
       stack.wsman.comm.socket.sendchannelclose()
       if (status === 200) {
         // log.info(`Power state change request successful for guid : ${uuid}`);
+        req.mpsService.mqtt.message({ type: 'success', method: 'AMT_PowerAction', guid: uuid, message: 'Sent Power Action' })
```

Comment:
Added the action code to the message. Turning this code into the proper string seems like a part of the despaghettification process rather than this pr